frontend/src/scenes/insights/Trends.cy-spec.js

```diff
@@ -0,0 +1,353 @@
+                            order: 0,
+                        },
+                    ]),
+                    properties: JSON.stringify([
+                        {
+                            key: '$browser',
+                            value: 'Chrome',
+                            operator: 'exact',
+                            type: 'event',
+                        },
+                        {
+                            key: '$current_url',
+                            value: 'http://posthog.com',
+                            operator: 'exact',
+                            type: 'event',
+                        },
+                    ]),
+                })
+
+            cy.get('[data-attr="trend-line-graph"]').should('be.visible')
+            cy.contains('http://posthog.com').should('be.visible')
+        })
+
+        it('responsive to shown as', () => {
+            helpers.setLocation('/insights', {
+                insight: 'TRENDS',
+                interval: 'day',
+                display: 'ActionsLineGraph',
+                events: [
+                    {
+                        id: '$pageview',
+                        name: '$pageview',
+                        type: 'events',
+                        order: 0,
+                    },
+                ],
+                properties: [],
+                shown_as: 'Stickiness',
+            })
+            mount()
+            cy.wait('@api_insight')
+                .map(helpers.getSearchParameters)
+                .should('include', {
+                    insight: 'TRENDS',
+                    interval: 'day',
+                    display: 'ActionsLineGraph',
+                    events: JSON.stringify([
+                        {
+                            id: '$pageview',
+                            name: '$pageview',
+                            type: 'events',
+                            order: 0,
+                        },
+                    ]),
+                    properties: '[]',
+                    shown_as: 'Stickiness',
+                })
+            cy.get('[data-attr="trend-line-graph"]').should('be.visible')
+            cy.contains('Stickiness').should('be.visible')
+        })
+
+        it('responsive to breakdown', () => {
+            helpers.setLocation('/insights', {
+                insight: 'TRENDS',
+                interval: 'day',
+                display: 'ActionsLineGraph',
+                events: [
+                    {
+                        id: '$pageview',
+                        name: '$pageview',
+                        type: 'events',
+                        order: 0,
+                    },
+                ],
+                properties: [],
+                breakdown: '$browser',
+                breakdown_type: 'event',
+            })
+            mount()
+
+            cy.wait('@api_insight')
+                .map(helpers.getSearchParameters)
+                .should('include', {
```

Comment:
Q: Is this 'include' or 'eq'? Seems like we've included just about everything.

Reply:
<img width="659" alt="Screen Shot 2021-02-11 at 2 39 22 PM" src="https://user-images.githubusercontent.com/13127476/107693628-61f32000-6c7c-11eb-8ba7-b6318adb46f2.png">
 
Any idea why the equality here isn't fulfilled?

Reply:
I think it might be using reference equality. 'eql' might be the right thing here, if not then chai - docs might hold the answer

Reply:
both eq and eql weren't working in that instance 🤔 but I also opted for you pattern for default mounting so that check is uneeded. I focused the includes to be specific on params that will be tested for